index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="index.css">
+    <script src="https://kit.fontawesome.com/0bd1156d30.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="Main-Page">
+     <nav class="nav">
+<div class="logo">AFIAMOAH</div>
```

Comment:
- [ ] What do you think about using an `a` tag instead of a `div` for your logo? In most websites the logo is a link that leads to the homepage so you should replace this logo div with an `a`.